ui/client/containers/paths.ts

```diff
@@ -11,4 +11,5 @@ export const SignalsPath = `/signals`
 export const ServicesPath = `/services`
 export const CustomTabBasePath = `/customtabs`
 export const ScenariosBasePath = `/scenarios`
+export const ArchivedPath = `${ScenariosBasePath}/?SHOW_ARCHIVED=true&HIDE_ACTIVE=true`
```

Comment:
wouldn't it work in both cases, I mean in old and new one ?

Reply:
Let's add comment that it don't work with old scenario list